### PR TITLE
Add info to CONTRIBUTING.md about external bugs

### PR DESCRIPTION
**Pull request by andrewsavage1**
b/261052420

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ Cobalt coding style.
 
 #### General guidelines and philosophy for contribution
 
+*   Ensure your change references the bug you are addressing. Follow the
```
Comment (kaidokert) [inline CONTRIBUTING.md:53]: Should we add the `references the bug in form b/124 on a separate commit line`, for now ? 

Obviously i want to make it easier and just require it in PR description soon, but that work still needs doing.

**Review by kaidokert — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (andrewsavage1) [reply CONTRIBUTING.md:53]: Can we just do that work? Isn't there already a PR with it?

**Review by alexanderahn — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kaidokert) [reply CONTRIBUTING.md:53]: No there isn't ye. We'd need to flip the checks over to verify only PR title and description, and ensure it works for all cases. I'm also expecting it to break gitwatcher, so we'd need an alternative.

Comment (andrewsavage1) [reply CONTRIBUTING.md:53]: This is all that's needed https://github.com/youtube/cobalt/pull/408
What do you expect to break with gitwatcher? The way squashed + merged commits are migrated won't change

**Pull request merged**